product-development/README.md

```diff
@@ -0,0 +1,83 @@
+# Product Development
+
+## Workflow
+We have a lightweight product develoment workflow that empowers individual Tilters to collaboratively solve business problems, work directly with customers, and to be accountable for achieving positive business outcomes.
+
+<img src="images/workflow.png" height="300" />
+
+The workflow is summarized in the image above. At the far left, we continually have conversations with customers and the community. These are synthesized into GitHub issues and Clubhouse stories. Based on this feedback and our broader company goals and strategy, we maintain a backlog of high-level initiatives as Clubhouse epics (primarily product changes). Over time, epics are further defined with scoped work and acceptance criteria, as they are eventually implemented and shipped.
+
+### Free-form discussion in Clubhouse stories
+All Tilters are encouraged to collaborate in _discussing_ general ideas in a free-form fashion, with no strict process, no structure, no specific timelines, and no specific implementation commitments. Tilters should prefer discussing ideas in Clubhouse stories, linking ideas together, thus building a web of good ideas that are easily searchable for posterity.
+
+### Defining high-level business initiatives with Clubhouse epics
+All Tilters are encouraged to collaborate in _definining_ high-level business initiatives. Similar to general ideas, there is no strict process for collaboration, except that each business initiative should be documented as a Clubhouse epic. As an epic becomes increasingly well-defined over time, it should have more and more of these components with increasing fidelity:
+- Problem statement
+- Solution scope
+- Attached stories with relevant ideas, including designs
+
+Victor is responsible for ensuring that the backlog of epics is continually full and that epics are increasingly being better defined. There should be no shortage of epics.
+
+## Picking an epic to work on
+When a Tilt engineer is free, they should pick a high priority epic to work on and make themself the _sole_ owner of the epic. The engineer should find other folks to collaborate on the epic as needed, but they are the [DRI](https://medium.com/@mmamet/directly-responsible-individuals-f5009f465da4) (directly responsible individual) for driving the epic to completion. There should be at most one owner on any epic, and the owner is the DRI.
+
+The epic may have a fairly broad initial business problem. The engineer(s) should carve out and define a smaller problem, and spec out a solution that can be accomplished within a target of **_7 working days_** (counted from when the epic moves into `In Progress`, more below on states). If the solution cannot be finished within 7 days, keep reducing scope. The epic description should be updated with the smaller scope and acceptance criteria, and the engineer(s) should create additional epics and/or stories to capture the future work.
```

Comment:
@nicks : I updated this to be "7 calendar working days". It's meant to be a time estimate for the entire epic, not an effort estimate for a single person. Do you think that makes sense? Or not?

Reply:
FWIW, when I read this, I found "calendar working days" confusing and wasn't sure if it was a typo, since "calendar days" and "working days" are kind of opposites (whether they include weekends), and the number 7 seems arbitrary enough that maybe it does mean a week of 7 calendar days.

I think "business days" would clearly communicate "working days" while being less confusable with "person days".

(I'd also update all the other "7 days" in the doc)

Reply:
Changed to "7 business days". Thanks @landism